lib/gollum/app.rb

```diff
@@ -382,9 +384,10 @@ class App < Sinatra::Base
         wikip     = wiki_page(params[:splat].first)
         @name     = wikip.fullname
         @page     = wikip.page
-        @page_num = [params[:page].to_i, 1].max
+        @page_num = [params[:page_num].to_i, 1].max
+        @max_count = settings.wiki_options.fetch(:latest_changes_count, 10)
```

Comment:
Why `:latest_changes_count` in `/history` and not something more generic like `pagination_take`?

Reply:
Thanks, I like `pagination_take`!

Reply:
Or `pagination_take_count` or `pagination_count`?

Reply:
`pagination_take_size`?